server/models/User.ts

```diff
@@ -54,6 +54,12 @@ export enum UserRole {
   Viewer = "viewer",
 }
 
+export enum UserPreference {
+  LastVisitedUrl = "lastVisitedUrl",
```

Comment:
Maybe we should store `lastVisitedPath` instead, without the host? It would come as handy while handling redirection and avoid unnecessary clutter.